src/modules/snd/libkvisnd.h

```diff
@@ -146,7 +149,10 @@ class KviSoundPlayer : public QObject
 	Phonon::MediaObject * m_pPhononPlayer;
 #endif //!COMPILE_PHONON_SUPPORT
 #ifdef COMPILE_QTMULTIMEDIA_SUPPORT
-	QSoundEffect * m_pSoundEffect;
+	QMediaPlayer * m_pMediaPlayer;
```

Comment:
I think new code shouldn't be using new/delete anymore, but unique_ptr

Reply:
I would prefer a separate commit updating the whole source file code to modern c++ standard

Reply:
How do you imagine writing or even reviewing such commit which gets rid of all the legacy code and replace everything with something more modern?

That's basically rewriting the whole kvirc from scratch :)

Gradually it's at least possible to do

Reply:
Gotcha. Done in [386e46e](https://github.com/kvirc/KVIrc/pull/2669/commits/386e46e3f6ce9d4c0d30ab1a19221e75a2966940)